### ISSUE TITLE
WARNING - The model and loaded state dict do not match exactly,

### ISSUE THREAD

**Issue by chenshaobo** [open-mmlab/mmdetection#1495]
Thanks for your error report and we appreciate it a lot.

**Checklist**
1. I have searched related issues but cannot get the expected help.
2. The bug has not been fixed in the latest version.

**Describe the bug**

**Reproduction**
1. What command or script did you run?
```
A placeholder for the command.
```
2. Did you make any modifications on the code or config? Did you understand what you have modified?
3. What dataset did you use?

**Environment**
 - OS: [e.g., Ubuntu 16.04.6]
 - GCC [e.g., 5.4.0]
 - PyTorch version [e.g., 1.1.0]
- How you installed PyTorch [e.g., pip, conda, source]
- GPU model [e.g., 1080Ti, V100]
- CUDA and CUDNN version
- [optional] Other information that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**
If applicable, paste the error trackback here.
```
A placeholder for trackback.
```

**Bug fix**
If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by chenshaobo** [comment]
WARNING - The model and loaded state dict do not match exactly
these keys have mismatched shape:
+-------------------------+------------------------+-------------------------+
| key                     | expected shape         | loaded shape            |
+-------------------------+------------------------+-------------------------+
| bbox_head.fc_cls.weight | torch.Size([16, 1024]) | torch.Size([81, 1024])  |
| bbox_head.fc_cls.bias   | torch.Size([16])       | torch.Size([81])        |
| bbox_head.fc_reg.weight | torch.Size([64, 1024]) | torch.Size([324, 1024]) |
| bbox_head.fc_reg.bias   | torch.Size([64])       | torch.Size([324])       |
+-------------------------+------------------------+-------------------------+

**Comment by hellock** [comment]
Your dataset has different number of classes from the model pretrained on COCO.

**Comment by klauscf** [comment]
> Your dataset has different number of classes from the model pretrained on COCO.

after i change the num_class from 81 to 21(change the dataset coco -->VOC),the same mistake happened again,there is the list about the problem
The model and loaded state dict do not match exactly

these keys have mismatched shape:
+-----------------------------+------------------------------+------------------------------+
| key                         | expected shape               | loaded shape                 |
+-----------------------------+------------------------------+------------------------------+
| bbox_head.retina_cls.weight | torch.Size([180, 256, 3, 3]) | torch.Size([720, 256, 3, 3]) |
| bbox_head.retina_cls.bias   | torch.Size([180])            | torch.Size([720])            |
+-----------------------------+------------------------------+------------------------------+
    (retina_cls): Conv2d(256, 180, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (retina_reg): Conv2d(256, 36, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
i wonder somewhere need to be changed too,but i don't what should i do

**Comment by HymEric** [comment]
I encountered the same error.
